app/settings.py

```diff
@@ -38,7 +38,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'helloworld'
+    'redcycle.apps.RedcycleConfig'
```

Comment:
I removed the `helloworld` app that was previously there and create a `redcycle` app